src/utils/helper.rs

```diff
@@ -42,6 +44,37 @@ pub mod Util {
     pub const SGINFO: &str = "/usr/bin/sginfo";
     pub const JBOD_EXPORTER: &str = "/usr/bin/prometheus-jbod-exporter";
 
+    /// Returns an enum with true or false if a directory is empty
+    ///
+    /// This function verify if the directory is empty.
+    ///
+    /// # Arguments
+    ///
+    /// * `path` - a string reference
+    pub fn is_folder_empty(path: &str) -> io::Result<bool> {
+        Ok(fs::read_dir(path)?.next().is_none())
+    }
+
+    /// Verify if sysclass folder exists or otherwise exit and
+    /// provide software alternatives to jbod-rs.
+    pub fn verify_sysclass_folder(path: &str) {
+        if is_folder_empty(path).unwrap_or(false) {
+            println!(
+                "{} {} {}",
+                "==> ".red().bold(),
+                "jbod-rs not supported in this machine".bold(),
+                " <==".red().bold()
+            );
+            print!("{}", ":: ".bold().red());
+            println!("Use one of these alternatives: ");
+            println!(" - lsscsi");
+            println!(" - sas-lsi-tools");
+            println!(" - sas2ircu-status");
+            println!(" - sg3-utils");
+            exit(1);
```

Comment:
Don't know if it's important: Here the exit error code changed from `exit(2) to `exit(1)` don't know if it's something important here.

Reply:
Before it was some like this:
https://github.com/Gandi/jbod-rs/pull/26/files#diff-94b75cd2f1af4fbd07ce930fb40350b7b86d02c4efabd1cb3ae434bf379638b4L459

Reply:
Before it was some like this:
https://github.com/Gandi/jbod-rs/pull/26/files#diff-94b75cd2f1af4fbd07ce930fb40350b7b86d02c4efabd1cb3ae434bf379638b4L459